Example `json_obj.go` failing to execute

Example code is failing with error -
```
➜  go-rejson git:(master) ✗ go run examples/json_obj.go
# command-line-arguments
examples/json_obj.go:82:4: undefined: rejson.JSONGetOptionIndent
examples/json_obj.go:82:39: undefined: rejson.JSONGetOptionNewLine
examples/json_obj.go:83:4: undefined: rejson.JSONGetOptionSpace
examples/json_obj.go:83:37: undefined: rejson.JSONGetOptionNoEscape
```

Code snippet -
```
json_obj.go
------------
...
	res, err = rejson.JSONGet(conn, "obj", ".",
		&rejson.JSONGetOptionIndent{"\t"}, &rejson.JSONGetOptionNewLine{"\n"},
		&rejson.JSONGetOptionSpace{" "}, &rejson.JSONGetOptionNoEscape{})
	if err != nil {
		log.Fatalf("Failed to JSONGet")
		return
	}
...
```

structs `JSONGetOptionIndent` do not exist in `rejson.go` and have been converted to constructors `NewJSONGetOptionIndent`. 

Convert this code to use the ctors to make it pass.

--- a/examples/json_obj.go
+++ b/examples/json_obj.go
@@ -78,8 +78,8 @@ func main() {
 	fmt.Println("Memory used by obj:", res)
 
 	res, err = rejson.JSONGet(conn, "obj", ".",
-		&rejson.JSONGetOptionIndent{"\t"}, &rejson.JSONGetOptionNewLine{"\n"},
-		&rejson.JSONGetOptionSpace{" "}, &rejson.JSONGetOptionNoEscape{})
+		rejson.NewJSONGetOptionIndent("\t"), rejson.NewJSONGetOptionNewLine("\n"),
+		rejson.NewJSONGetOptionSpace(" "), rejson.NewJSONGetOptionNoEscape())
 	if err != nil {
 		log.Fatalf("Failed to JSONGet")
 		return

